server/handlers/database_handlers.go

```diff
@@ -116,20 +116,10 @@ func (h *Handler) GetSystemDatabase(w http.ResponseWriter, r *http.Request, _ *m
 // - Applies auto migration to recreate the necessary tables.
 //
 // responses:
-//   200: successfulResponse
-//   500: errorResponse
-
-// successfulResponse: database reset successful
-// A successful response indicating that the database reset operation was successful.
-// ---
-// type: string
-// example: "Database reset successful"
-
-// errorResponse: error response
-// An error response indicating the reason for the failure of the database reset operation.
-// ---
-// type: string
-// example: "Directory could not be created due to a non-existent path."
```

Comment:
@Aritra8438 is this accurate?

Reply:
Yes @leecalcote. There might be many reasons that the databse reset is not successful. 
"Directory could not be created due to a non-existent path." is one of the possibilities.

Reply:
Hi, @leecalcote, I think the problem is in `200: successfulResponse` considering the issue you mentioned.

I've already mentioned the successfulResponse after this, but that should be mentioned at `handlers.go`.

Should I link a more definite PR?